poetry/core/packages/package_specification.py

```diff
@@ -0,0 +1,118 @@
+from typing import FrozenSet
```

Comment:
@sdispater one minor request can we rename this to `poetry.core.packages.specification` please?

Reply:
@abn You only want to the module name to be changed, right? Not the name of the class? `poetry.core.packages.specification` would provide the `PackageSpecification`.

Reply:
Correct. Feels a bit cleaner.

Reply:
Done :-)